var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -128,6 +135,38 @@ def cmake_args(self):
                    '-DDISABLE_NRN_TIMEOUT=ON'
                    ]
 
+        if spec.satisfies('+profile'):
+            env['CC']  = 'tau_cc'
+            env['CXX'] = 'tau_cxx'
+        elif 'bgq' in spec.architecture and spec.satisfies('+mpi'):
+            env['CC']  = spec['mpi'].mpicc
+            env['CXX'] = spec['mpi'].mpicxx
+
+        if spec.satisfies('+nmodl'):
+            options.append('-DENABLE_NMODL=ON')
+            options.append('-DNMODL_ROOT=%s' % spec['nmodl'].prefix)
+            flags += ' -I%s -I%s' % (spec['nmodl'].prefix.include, spec['eigen'].prefix.include.eigen3)
```

Comment:
@ohm314 : aren't these options now changed in the master branch?

Reply:
They are, but this only affects `develop`, the fixes will come in with my branch. Just need to merge in order :)

Reply:
ok understood then.